GitHubRepositories.ps1

```diff
@@ -131,6 +151,24 @@ function New-GitHubRepository
         'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
     }
 
+    if ($PSBoundParameters.ContainsKey('Private'))
+    {
+        Write-Log -Level Warning -Message ('The Private switch is being replaced by the Visibility parameter.  ' +
```

Comment:
This message also got me thinking..._we_ have scripts/usage that needs to migrate as well.  Can you also update the `.EXAMPLES` in the CBH within this script as well as the few usages of `-Private` in the tests (I found two instances)?

Reply:
Changed examples and one reference in `GitHubRepositories.tests.ps1`. Couldn't see a second reference in the tests?

Reply:
Yeah, the other reference was `Set-GitHubProject` ... I didn't look at the code carefully enough.